suggestions/service.py

```diff
@@ -0,0 +1,87 @@
+import abc
+import logging
+
+from mailjet_rest import Client
+
+from .model import Suggestion
+
+
+class SuggestionService(abc.ABC):
+    """
+    Service to send a suggestion
+    """
+
+    @abc.abstractmethod
+    def add_suggestion(self, suggestion: Suggestion):
+        """
+        Method to send a suggestion
+        """
+
+
+class MailjetSuggestionService(SuggestionService):
+    def __init__(
+        self,
+        mailjet_client: Client,
+        suggestion_email_sender: str,
+        suggestion_email_recipient: str,
+        suggestion_name: str,
```

Comment:
Should not `suggestion_email_sender` and `suggestion_name` come from the users? Why do we need to define this in the `__init__` method?

Reply:
Actually, this was quite confusing for us too. Hope I can clarify things now, hehe:

Mailjet needs a verified sender. The recipient can be any email. For us, both the recipient and sender should be known because we would be sending "from us, to us". Therefore, they should be envvars.

The user will send us a name and email too (`suggestion.name` and `suggestion.email_address`), but since it isn't used to send or receive an email, it is sent in the email body in case we want to send a reply.